files/query_cache.py

```
@@ -0,0 +1,354 @@
+            # Also clear related media for other videos that might reference this one
+            f"{CACHE_KEY_PREFIX}:related_media:*",
+        ]
+
+        total_deleted = 0
+        for pattern in patterns:
+            count = cache.delete_pattern(pattern, version=QUERY_CACHE_VERSION)
+            total_deleted += count
+            logger.debug(f"Invalidated {count} cache entries for pattern: {pattern}")
+
+        logger.info(f"Invalidated {total_deleted} cache entries for media {friendly_token}")
+        return total_deleted
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)